src/platform/storage/zephyr/nvs/storage.c

```diff
@@ -159,6 +159,9 @@ mender_storage_init(void) {
         mender_log_error("Unable to mount NVS storage, result = %d", result);
         return MENDER_FAIL;
     }
+    mender_log_debug("Initialized Mender NVS with %u sectors (%zd bytes available)",
```

Comment:
`sector.size` is `uint16_t`, not `size_t`, so this should be `%u bytes` (?)

Reply:
I think it's better to add `(size_t)` in front of the multiplication.